authentication-typescript/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.1.2 (July 21, 2021)
+
+### Added
```

Comment:
Can you also add a Changed section to mention that we are locking on MUI v4?

### Changed

-   Lock Material UI dependency on version 4.11 to avoid v4 -> v5 deprecation warnings. We will have new templates in the future that will support v5.
-   React Strict Mode no longer enabled by default.